plugins/@grouparoo/postgres/__tests__/table-import/import-properties.ts

```diff
@@ -882,96 +918,14 @@ describe("postgres/table/recordProperties", () => {
           },
           [{ op, key: "amount", match: "1.54" }]
         );
-        expect(values[record.id][properties[0].id]).toEqual([4]);
+        expect(values[record.id][properties[0].id]).toEqual([5]);
         expect(values[otherRecord.id][properties[0].id]).toEqual([4]);
         expect(values[thirdRecord.id][properties[0].id]).toEqual([0]);
       });
     });
 
-    describe("equals", () => {
-      const op = "equals";
```

Comment:
Why was this section removed?

Reply:
It is a duplicate of the section starting on line 599